include/aca_net_state_handler.h

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 The Alcor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ACA_NET_STATE_HANDLER_H
+#define ACA_NET_STATE_HANDLER_H
+
+#include "aca_dataplane_mizar.h"
+#include "goalstateprovisioner.grpc.pb.h"
+
+namespace aca_net_state_handler
+{
+class Aca_Net_State_Handler {
```

Comment:
here are the actual interfaces?

Reply:
do we allow existence of multiple handlers?

Reply:
this is the header file for network state handler, there is only one handler implemented as singleton class. It can invoke one or more dataplane implementation as long as the dataplane class implements ACA_Core_Net_Programming_Interface defined in aca_net_programming_if.h, e.g.:
-  class ACA_Dataplane_Mizar : public ACA_Core_Net_Programming_Interface
-  class ACA_Dataplane_OVS : public ACA_Core_Net_Programming_Interface (to be implemented)